src/components/page/page_body/page_body.tsx

```diff
@@ -61,6 +62,7 @@ export type OuiPageBodyProps<T extends ComponentTypes = 'main'> = CommonProps &
     panelled?: boolean;
     /**
      * Extends any extra OuiPanel props if `panelled=true`
+     * **panelled prop is deprecated in version 2.0.0
```

Comment:
This is comment is on the wrong line - comment precede the props, so the comments for the `panelled` prop are L59-61.